README.md

```diff
@@ -22,7 +28,63 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+The current main usage of `FastStats` is to track and build means while iterating over data sets.
+
+Example: Let's say you want to get some means for social media posts.
+
+You can track several metrics with `FastStats::Means`:
+
+```ruby
+# Say you have some PostFetcher that is an enumerator for your posts.
+means = FastStats::Means.new
+
+PostFetcher.each do |post|
+   # do stuff with post
+   means.add "likes", post["like_count"]
+   means.add "comments", post["share_count"]
+end
+
+means.summary
+# =>
+#{
+#   "foo_arithmetic"=>4.888888888888889,
+#   "foo_geometric"=>3.345423581422162,
+#   "bar_arithmetic"=>13.636363636363637,
+#   "bar_geometric"=>7.1824970648723765
+#}
+
+# You can also pass a round: value
+means.summary round: 2
```

Comment:
Does `summary` take any other args?

Reply:
Nope - do you think that should be said explicitly?

Reply:
Yes.